RegistrarApi/Controllers/SettingController.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Threading.Tasks;
+using System.Web.Http;
+using Common;
+using Common.Exceptions;
+using Registrar.Database.Interfaces;
+using Registrar.Models.Request;
+
+namespace Registrar.Api.Controllers
+{
+    [RoutePrefix("settings")]
+    public class SettingController : ApiController
+    {
+        private readonly IRegiSettingStore _store;
+
+        public SettingController(IRegiSettingStore store)
+        {
+            _store = store;
+        }
+
+        [HttpPost]
+        [Route("")]
+        [ApiKeyAuth]
+        public async Task<IHttpActionResult> Post(UpdateRegiSetting setting)
+        {
+            try
+            {
+                var updatedSetting = await _store.UpdateSetting(setting);
+                return Json(updatedSetting);
+            }
+            catch (RecordNotFoundException)
+            {
+                return NotFound();
+            }
+        }
+
+        [HttpGet]
+        [Route("list")]
+        [ApiKeyAuth]
+        public async Task<IHttpActionResult> List()
+        {
+            try
+            {
+                var settings = await _store.ListSettings();
+                return Json(settings);
+            }
+            catch (RecordNotFoundException)
+            {
+                return NotFound();
```

Comment:
Or just don't throw an exception in the Store, given that just because there are no settings, it doesn't necessarily indicate a problem